apps/passport-server/src/database/queries/telegram/fetchTelegramEvent.ts

```diff
@@ -154,3 +160,60 @@ export async function fetchUserTelegramChats(
   );
   return result.rows[0] ?? null;
 }
+
+export type ChatIDWithEventsAndMembership = ChatIDWithEventIDs & {
+  isChatMember: boolean;
+};
```

Comment:
these types, as well as all other types and interfaces in this file, should go in `models.ts`